modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -8,11 +8,26 @@ endif::[]
 . Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Download Site].
 . Extract the Windows installation file into the `C:\` root folder.
 ** Ensure that your user has read and write permissions for the `C:\` directory.
-** If you are using a Windows antivirus application, ensure that the `plugins/` and `features/` directories are under the _trusted_ category.
+** If you are using a Windows antivirus application, ensure that the `plugins/` and `features/` directories are under the *trusted* category.
 . Open the extracted Anypoint Studio.
 +
 When Studio first runs, the embedded OpenJDK Platform libraries might cause a Windows Security alert. +
 Ensure to allow the OpenJDK Platform library from Temurin publisher through your Windows firewall.
-. Click OK to accept the default workspace. +
+. Click *OK* to accept the default workspace. +
 ** If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or later. +
 Studio 7.x supports only Mule 4.x projects, because the structure of the project, export format, XML, and scripting language are different than the ones used by Mule 3.x.
+
+== Exclude Studio from Windows Defender Scans
+
+While Windows Defender is a tool for keeping your computer safe, it can also slow down the performance of Studio because it scans files in real-time as they are accessed, which causes delays. To prevent performance issues, you can exclude Studio from Windows Defender scans. 
+
+. Configure Windows Defender to scan your machine in real-time.
+. Run Studio. The following window pop-ups to ask if Studio folders and processes should be excluded:
++
+image::windows-defender-exclusions.png[“Windows Defender exclusions popup asks the user if folder and processes should be excluded.”]
```

Comment:
The pop-up image should be changed by another, it might be taken from the RC.

Reply:
@mauriarroqui, can you share the popup image from RC when available?
If not, who may help me with this SS? Thanks!

Reply:
Thanks for the SS! ;)